doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -3,161 +3,177 @@
+
 
-4) Run the resulting executable:
+4) Run the resulting executable, which will print 100 messages in parallel:
 
    .. code-block:: bash
 
-      ./hello
+      ./hello3-datapar
+
 
 5) Experiment with Chapel in this Quickstart mode to your heart's
-   content.  Once you are comfortable with Chapel and interested in
+   content.  If you'd like to use this build of Chapel in a different
+   shell / terminal session, see :ref:`using-chapel-in-another-shell`
+   below.  Once you are comfortable with Chapel and interested in
    using a full-featured version in the preferred configuration, see
    the next section.
 
 
 .. _using-a-more-full-featured-chapel:
 
-Using a More Full-Featured Chapel
----------------------------------
+Using Chapel in its Preferred Configuration
+-------------------------------------------
 
-To use Chapel in a more full-featured and preferred configuration,
-you will need to rebuild Chapel from source in a different configuration.
-
-*  Open up a new shell to avoid inheriting the previous environment
-   settings.
-
-*  The quickstart configuration described above sets ``CHPL_LLVM=none``
-   but this is not the preferred configuration of Chapel. As of 1.25,
-   LLVM is the default compiler backend and it needs to be available
-   for full functionality. So make sure you have a compatible system LLVM
-   installed in a standard location or set the environment variable
-   ``CHPL_LLVM`` to ``bundled`` or ``none`` as described in
-   :ref:`chplenv.rst <readme-chplenv.CHPL_LLVM>`.  If desired, you must
-   explicitly opt-out of using it by setting
-   ``CHPL_LLVM=none``. The ``CHPL_LLVM=bundled`` build takes a long
-   time, so to avoid surprises you need to explicitly opt-in to using it.
-
-*  Repeat steps 2-5 above, but in Step 2, source ``util/setchplenv.bash``
-   instead of ``util/quickstart/setchplenv.bash``.
-   This will set up your environment to use Chapel in the preferred
-   configuration.  Building this configuration involves compiling
-   third-party packages, which will increase the overall build time.
-   If you run into any portability issues, please let us know via
-   :ref:`bugs.rst <readme-bugs>`.
+To use Chapel in its preferred, full-featured mode, you will need to
+rebuild Chapel from source in a different configuration:
 
-   .. code-block:: bash
+* Open up a new shell to avoid inheriting the previous environment
+  settings.
 
-      # Set environment variables to preferred configuration
-      source util/setchplenv.bash
+* The Quickstart configuration described above sets ``CHPL_LLVM=none``
+  for simplicity and to save time.  This causes the Chapel compiler to
+  use its C back-end, which is not the preferred option; as of Chapel
+  1.25, LLVM is the default back-end, which needs to be available for
+  full functionality.  There are a few options for using LLVM:
 
-      # re-build Chapel
-      make
+  - ensure that you have a version of LLVM 11 installed on your system
```

Comment:
@e-kayrakli : This is the "embedding a version number" that I feel more nervous will fall out of date.  But similar to the previous argument, I fear that not saying it will cause people to say "Hey, I have LLVM, but it didn't work, what gives?"